devel/xcode-build-server/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        SolaWing xcode-build-server 1.0.0 v
+name                xcode-build-server
+version             1.0.0
+revision            0
+platforms           {darwin}
+license             MIT
+categories          devel python
+maintainers         {woolsweater.net:macports @woolsweater} \
+                    openmaintainer
+supported_archs     noarch
+
+homepage            https://github.com/SolaWing/xcode-build-server
+
+description         Build Server Protocol implementation for Xcode projects
+
+long_description    ${name} integrates Xcode with Apple's sourcekit-lsp. \
+                    sourcekit-lsp doesn't itself support Xcode projects, but it does \
+                    provide a Build Server Protocol client to work with other \
+                    build systems. xcode-build-server implements a build \
+                    server to provide sourcekit-lsp with the build \
+                    information it needs for an Xcode project.
+
+checksums           rmd160  fd21bf312b47c15fc28d03b80de1ec39d76140a2 \
+                    sha256  08ea2b9a892670bb7c2f8ff90a97ae0be519744f42ba713bb1a56e247c77e885 \
+                    size    18673
+
+python.default_version      39
```

Comment:
Our default version of python now is 3.12, can this package use it?

Reply:
Yes, that should be fine. I included this because it's the lowest version that `xcode-build-server` says it supports; I wasn't sure how MacPorts preferred to handle that. I will change this.

Reply:
Removed.